content/tutorial-regular-keys.md

```diff
@@ -0,0 +1,764 @@
+      "tx_json" : {
+         "Account" : "rUAi7pipxGpYfPNg3LtPcf2ApiS8aw9A93",
+         "Fee" : "10",
+         "Flags" : 2147483648,
+         "Sequence" : 3,
+         "SigningPubKey" : "03AEEFE1E8ED4BBC009DE996AC03A8C6B5713B1554794056C66E5B8D1753C7DD0E",
+         "TransactionType" : "AccountSet",
+         "TxnSignature" : "30450221008F44064A901C3A170A046673EFC8091F2C409E1DB7D78A5EF0D18E9D868AE21002207F594F3BF3821E6DF64CD283F590F6578F809DFD1CAAF654F187645A7CAB91C4",
+         "hash" : "BED704F2A6A80DA4060446C95802F205EE41977A2F63F10B1F8952FC6D01B35B"
+      }
+   }
+}
+```
+
+<!-- MULTICODE_BLOCK_END -->
+
+
+### 4. Explore Next Steps
+
+Now that you're familiar with the benefits of assigning a regular key pair to an account, consider taking a look at these related topics and tutorials:
+
+* [How to Multi-Sign](tutorial-multisign.html)
+* [Issuing and Operational Addresses](concept-issuing-and-operational-addresses.html)
+* [Listing XRP as an Exchange](tutorial-listing-xrp.html)
+
+
+## Removing or Changing a Regular Key Pair
+
+If your account's regular key pair is compromised, or if you just want to periodically change the regular key pair as a security measure, use the [`SetRegularKey`](reference-transaction-format.html#setregularkey) method to remove or change the regular key pair for your account.
+
+The steps to change your existing regular key pair are almost the same as the steps to [assign a regular key](#assigning-a-regular-key-pair) for the first time. You generate the key pair and assign it to your account as a regular key pair, overwriting the existing regular key pair. However, the main difference is that when changing the existing regular key pair, you can use the existing regular private key to replace itself, whereas when assigning a regular key pair to an account for the first time, you have to use the account's master private key to do it.
+
+For more information about master and regular key pairs, see [Cryptographic Keys](concept-keys.html).
+
+If you want to simply remove a compromised regular key pair from your account, you don't need to generate a key pair first. Just use the [`SetRegularKey`](reference-transaction-format.html#setregularkey) method, omitting the `RegularKey` value in the request. Note that the method fails if you don't have another way of signing for your account currently enabled (either the master key pair or a [signer list](reference-transaction-format.html#multi-signing)).
+
+### Request Format
+
+Here's example of the request format that removes an existing regular key pair from an account. Note that it omits the `RegularKey` field.
+
+<!-- MULTICODE_BLOCK_START -->
+
+*WebSocket*
+
+```
+{
+   "command": "submit",
```

Comment:
This tutorial should match the previous one in terms of having signing and submitting separate. It's hardly more secure to submit in one stage with a regular key than it is to do with a master key. (The chance of being compromised is the same; the consequences of being compromised might be less severe.)

Reply:
GREAT! Thank you for this clarification - makes sense. I updated the "Verify the Regular Key Pair" tutorial to match as well.